kind/kind.sh

```diff
@@ -0,0 +1,123 @@
+#/bin/sh
+
+KUMACTL_BIN=${HOME}/bin/kumactl
+KUMA_VERSION="0.4.0"
+
+KIND_BIN=${HOME}/bin/kind
+KIND_VERSION="v0.8.0"
```

Comment:
need to add a check of the version and force an upgrde

Reply:
Done also for kumactl.